scalableminds/webknossos
# Added optional skipping of adding dataset in onboarding

This PR makes it optional whether the onboarding includes the prompt to upload an own dataset. This is done by setting the `skipAddingDatasetsInOnboarding` in `application.conf` to `true`.
### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Set the `skipAddingDatasetsInOnboarding` feature flag to true.
- Logout and got to `<base-url>/onboarding`.
- The onboarding should no longer include the third page that asks the user to upload her/his own dataset.

### Issues:
- fixes #3972

------
